README.md

```diff
@@ -71,3 +71,9 @@ $(eval $(call BuildPackageGluon,ffXX-hello-world))
 
 After creating the package, open a pull-request to the community-repository.
 
+
+## If you have commit access (in addition to the [Contributing Guidelines](CONTRIBUTING.md))
+
+* Do NOT use `git push --force`.
+* Use Pull Requests if you are unsure and to suggest changes to other developers.
+* Don't use merge commits if you accept a single-commit PR, do a "rebase"-merge instead. This will prevent the commit-history to be cluttered with superfluous "merge" logmessages
```

Comment:
I've disabled merge commits, so this should be rephrased. Only squash and rebase are now possible.